java/code/src/com/redhat/rhn/domain/formula/FormulaFactory.java

```diff
@@ -149,6 +149,28 @@ public static String getServerDataFile() {
         return dataDir + SERVER_DATA_FILE;
     }
 
+    /**
+     * Return a warning message in case some folder doesn't exist or have wrong access level.
+     * @return a warning message if cannot access one folder. NULL if all folder are ok.
+     */
+    public static String getWarningMessageAccessFormulaFolders() {
+        String message = "System folder(s) from salt-formula are unreachable. " +
+                "Check if folders(s) exists and have the correct permissions (755):";
```

Comment:
Please move this message to StringResource_en_US.xml in branding.

Reply:
Thank's Abid, just change it